pyapr/data_containers/PyParticleData.hpp

```diff
@@ -474,9 +498,13 @@ void AddPyParticleData(pybind11::module &m, const std::string &aTypeString) {
     std::string typeStr = aTypeString + "Particles";
     py::class_<TypeParticles, BaseParticles>(m, typeStr.c_str(), py::buffer_protocol())
             .def(py::init())
-            .def(py::init([](uint64_t num_particles) { return new TypeParticles(num_particles); }))
-            .def(py::init([](APR& apr) { return new TypeParticles(apr); }))
-            .def(py::init([](py::array_t<DataType, py::array::c_style | py::array::forcecast>& arr){ return new TypeParticles(arr); }))
+            .def(py::init([](uint64_t num_particles) { return TypeParticles(num_particles); }))
+            .def(py::init([](APR& apr) { return TypeParticles(apr); }))
+            .def(py::init([](py::array_t<DataType, py::array::c_style | py::array::forcecast>& arr){ return TypeParticles(arr); }))
+            .def(py::init([](PyParticleData<float>& other){ return TypeParticles(other); }))
+            .def(py::init([](PyParticleData<uint8_t>& other){ return TypeParticles(other); }))
+            .def(py::init([](PyParticleData<uint16_t>& other){ return TypeParticles(other); }))
+            .def(py::init([](PyParticleData<uint64_t>& other){ return TypeParticles(other); }))
```

Comment:
whats behind the other wording here?

Reply:
what do you mean?

Reply:
Simply the use of the word "other".. may i'm not getting the context.

Reply:
So this is binding constructors for PyParticleData from other PyParticleData objects, of the listed types. Maybe `particlesToCopy` or something would be more informative